### ISSUE TITLE
Doubt about expiration of json files

### ISSUE THREAD

**Issue by hc2acc** [udacimak/udacimak#161]
When I download the json files of my nanodegree, can I render them even after my nanodegree has expired? In other words, can I download json files of a few nanodegrees and render them in the future only when I need them, even if my nanodegree has expired by then?

**Comment by update-udaci** [comment]
Yes you can do so.I did the same thing and saved jason files of few nanodegree program.You can download data even after your subscribtion expires.
